### DISCUSSION TITLE
How is the i3 status line configured the README screenshot?

### DISCUSSION THREAD

**Discussion by rafalstapinski** [NTBBloodbath/doom-nvim#429] [Q&A] [answered]
Apologies for being rather off-topic from Doom Nvim itself, but does anyone know how/what is used to configure the statusline/status bar in the [screenshot](https://camo.githubusercontent.com/b67661feef4be699d486634288bffd5c58b61b738b0a620cc6d9776f34a33c82/68747470733a2f2f692e696d6775722e636f6d2f656a456e6c45502e706e67) on the README? I'm assuming this is using i3 window manager. If so, how/what is used to customize the top and bottom status bars in the desktop environment to show things like the CPU/RAM usage, time, volume, etc within colored icons? 

I'm fairly new to i3 within Arch and was interested in tweaking my DE and this looked like a really good looking setup.

**Comment by NTBBloodbath** [comment, marked as answer]
Hey, it was polybar in AwesomeWM in my very old dotfiles (you can find the repository in my profile). I don't remember the revision commit so it's going to be like looking for a needle in a haystack lol

**Reply by rafalstapinski** [reply]
sweet, thanks! guess i'm going digging
